pkg/restic/config_test.go

```diff
@@ -130,5 +142,40 @@ func TestGetRepoIdentifier(t *testing.T) {
 			},
 		},
 	}
-	assert.Equal(t, "gs:bucket-2:/prefix-2/restic/repo-2", GetRepoIdentifier(backupLocation, "repo-2"))
+	id, err = GetRepoIdentifier(backupLocation, "repo-2")
+	assert.NoError(t, err)
+	assert.Equal(t, "gs:bucket-2:/prefix-2/restic/repo-2", id)
+
+	backupLocation = &velerov1api.BackupStorageLocation{
+		Spec: velerov1api.BackupStorageLocationSpec{
+			Provider: "unsupported-provider",
+			StorageType: velerov1api.StorageType{
+				ObjectStorage: &velerov1api.ObjectStorageLocation{
+					Bucket: "bucket-2",
+					Prefix: "prefix-2",
+				},
+			},
+		},
+	}
+	id, err = GetRepoIdentifier(backupLocation, "repo-1")
+	assert.EqualError(t, err, "cannot determine restic repository prefix for backup storage location")
```

Comment:
This is currently failing tests:

```--- FAIL: TestGetRepoIdentifier (0.00s)

    config_test.go:161: 
        	Error Trace:	config_test.go:161
        	Error:      	Error message not equal:
        	            	expected: "cannot determine restic repository prefix for backup storage location"
        	            	actual  : "restic repository prefix (resticRepoPrefix) not specified in backup storage location's config"
        	Test:       	TestGetRepoIdentifier```

L88 of config.go seems to be a mismatch.

Reply:
oh yeah, changed this and forgot